.github/workflows/deploy-storybook.yml

```diff
@@ -0,0 +1,21 @@
+name: Deploy Storybook
+
+on:
+  push:
+    branches: [main]
+    paths: ["webapp/client/**"]
```

Comment:
I think it is fine like that. However, wouldn't "webapp/client/src/**" and "webapp/client/public/**" be enough?

Reply:
Hard to say what's the minimal set here. Your suggestion would e.g. ignore any changes to dependencies in `webapp/client/package.json`. I wanted to be safe and rather have a build too many than having to jump in to make a fix because something doesn't show up as expected, or worse: confuse someone because storybook and staging are different because of the deployment config. Besides: I'd say the instances where we make changes to the `client/` folder without it affect the storybooked components are pretty rare.